layout/_partial/sidebar-navigation.ejs

```diff
@@ -89,7 +89,7 @@
                 <% if(theme.sidebar.pages[i].icon){ %>
                     <i class="material-icons sidebar-material-icons"><%= theme.sidebar.pages[i].icon %></i>
                 <% } %>
-                <%= i %>
+                <%= __('sidebar.' + i) %> 
```

Comment:
is this will restrict the order of pages name in sidebar? or it will auto match the pages